modules/ROOT/pages/6.0.2-release-notes.adoc

```diff
@@ -0,0 +1,62 @@
+= TinyMCE 6.0.2
+:navtitle: TinyMCE 6.0.2
+:description: Release notes for TinyMCE 6.0.2
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.0.2 was released for {enterpriseversion} and {cloudname} on Wednesday, May 4^th^, 2022. It includes {productname} 6.0.2 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.0.2, including:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:general-bug-fixes[General bug fixes]
+* xref:known-issues[Known issues]
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.0.2.
+
+=== Spell Checker Pro 3.0.2
+
+The {productname} 6.0.2 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 3.0.2 fixes the page focus incorrectly changing when spellchecking is triggered on multiple editors after scrolling.
```

Comment:
```suggestion
In the previous release, **Spell Checker Pro** would change page focus incorrectly when spellchecking was triggered on multiple editors after a `scroll` event.

**Spell Checker Pro** 3.0.2, which accompanies this release, corrects this so that the page focus no longer moves in this circumstance.
```